Development.podspec

```diff
@@ -0,0 +1,91 @@
+Pod::Spec.new do |s|
+  s.name         = 'Development'
+  s.version      = '0.1.0'
+  s.summary      = 'Infrastructure for Workflow-powered UI'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { :git => 'https://github.com/square/workflow.git', :tag => "v#{s.version}" }
+
+  s.ios.deployment_target = '10.0'
+  s.swift_version = '5.0'
+  s.dependency 'Workflow'
+  s.dependency 'WorkflowUI'
+  s.source_files = 'swift/Samples/Dummy.swift'
+
+  s.subspec 'Dummy' do |ss|
+  end
+
+  s.default_subspecs = 'Dummy'
+
+  dir = Pathname.new(__FILE__).dirname
+
+  s.scheme = { 
+      environment_variables: { 
+        'FB_REFERENCE_IMAGE_DIR' => dir.join('swift/Samples/SplitScreenContainer/SnapshotTests/ReferenceImages'),
+        'IMAGE_DIFF_DIR' => dir.join('swift/Samples/SplitScreenContainer/SnapshotTests/FailureDiffs'),
+      }
+    }
+
+  s.app_spec 'SampleApp' do |app_spec|
+    app_spec.source_files = 'swift/Samples/SampleApp/Sources/**/*.swift'
+    app_spec.resources = 'swift/Samples/SampleApp/Resources/**/*.swift'
+    app_spec.pod_target_xcconfig = {
+      'INFOPLIST_FILE' => '${PODS_ROOT}/../../swift/Samples/SampleApp/Configuration/Info.plist'
```

Comment:
What's this all about?

Reply:
There was already an existing plist for this sample - we can let `pod gen` create one for us instead if its not needed, but I had added this because one of the samples needed the iOS 13 scene delegation that was specified in their plist (I believe it was the SwiftUI one that isn't supported ATM)

Reply:
I'll double check and remove this if it isn't necessary

Reply:
This file looks to be deleted?

Reply:
Yep, good catch